modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/Export.tsx

```diff
@@ -13,6 +13,7 @@
  */
 
 import ClayTable from '@clayui/table';
+import dayjs from 'dayjs';
```

Comment:
Use this import instead, the ../../ may be different

We have **dayjs** configured there

```suggestion
import dayjs from '../../util/date';
```